Rakefile

```diff
@@ -92,7 +93,9 @@ namespace :test do
 end
 
 desc 'Lint the code through pylint'
-task 'lint' => 'ci:default:lint'
+task 'lint' => ['ci:default:lint'] do
+  sh 'rubocop --fail-level W'
```

Comment:
Instead of `--fail-level W`, could you instead create a `.rubocop.yml` file to only ignore specific errors ? You can generate one with `rubocop --auto-gen-config`.